rechunker/api.py

```diff
@@ -142,12 +142,13 @@ def _encode_zarr_attributes(attrs):
 
 def _zarr_empty(shape, store_or_group, chunks, dtype, name=None, **kwargs):
     # wrapper that maybe creates the array within a group
-    if name is not None:
-        assert isinstance(store_or_group, zarr.hierarchy.Group)
+    if isinstance(store_or_group, zarr.hierarchy.Group):
```

Comment:
Is there a difference between `zarr.hierarchy.Group` and `zarr.Group`? Later on you use the latter.

Reply:
No difference.